packages/spright-components/src/rectangle/styles.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderColor,
+    borderWidth
+} from '@ni/nimble-components/dist/esm/theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        border: ${borderWidth} solid ${borderColor};
```

Comment:
nit / pref: lets use borderColorHover just to give it some pop

Reply:
Yep. I made that same change in my other branch where I am adding the Storybook. I also added some other styling (hover, disabled) to make the stories a bit more educational, but we can discuss that in the next PR.

Reply:
I guess so, but like I started [here](https://github.com/ni/nimble/pull/1986#discussion_r1572889302) I'm not looking to maintain this component as an example and treating it as something to delete when we get a real component. It's just to help setup infra.